### PR TITLE
Update examples list

### PR DESCRIPTION
**Pull request by astefanutti**

### CODE REVIEW

File: examples/stable-diffusion-dreambooth/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-This example demostrate how to fine tune Stable Diffusion with DreamBooth and Ray Train using OpenShift AI.  
+This example demonstrate how to fine tune Stable Diffusion with DreamBooth and Ray Train using OpenShift AI.
```
Comment (Fiona-Waters) [inline examples/stable-diffusion-dreambooth/README.md:3]: Nit :) 
```suggestion
This example demonstrates how to fine tune Stable Diffusion with DreamBooth and Ray Train using OpenShift AI.  
```

Comment (astefanutti) [reply examples/stable-diffusion-dreambooth/README.md]: Thanks, I've actually prepared another PR as there are more typos to fix in that file. I've reverted that change here and will create a follow up PR.

Comment (Fiona-Waters) [reply examples/stable-diffusion-dreambooth/README.md]: Sounds good, we can merge this then. Thanks

**Review by Fiona-Waters — APPROVED** [review]
/lgtm
/approve

**Pull request merged**